docs/src/security/transparency/2021_report.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "2021 Platform.sh Transparency Report"
+sidebarTitle: "2021 Report"
+file: "/files/reports/transparency-abuse/2021_platformsh_transparency_report.pdf"
+---
+
+{{< description >}}
```

Comment:
The description might benefit from a list.

From:
> Platform.sh provides two reports on an annual basis according to the EU Digital Services Act Package, French law, and European Data Protection Board’s requirements and recommendations outlining transparency and abuse during that year. The contents of that report are listed below, and can also be downloaded as a PDF.

To:

Platform.sh provides two reports on an annual basis according to the:
- [EU Digital Services Act Package](https://digital-strategy.ec.europa.eu/en/policies/digital-services-act-package), 
- French law,
- [European Data Protection](https://gdpr.eu/what-is-gdpr/),
- Board’s requirements and recommendations outlining transparency and abuse during that year. 

The contents of that report are listed below, and can also be downloaded as a PDF.

Reply:
The links would need to be reviewed by the compliance team. Regarding french law, I was not sure which would be the exact article.

Reply:
The aim here isn't to provide comprehensive technical documentation. I think a list would be somewhat out of place here. Linking to the general GDPR also wouldn't be so helpful in understanding the specific focus on the board in this area.